src/late_multicellular_stage/Scalis.cs

```diff
@@ -36,6 +46,26 @@ public Scalis(IReadOnlyCollection<MulticellularMetaball> metaballs)
             points[i] = point;
             ++i;
         }
+
+        boneMaxSquareDistanceCache = new float[points.Length];
+
+        for (int j = 0; j < points.Length; j++)
```

Comment:
Can't this loop reuse the i variable so that this then looks like a simple "for i" loop?

`        for (i = 0; i < points.Length; ++i)`

Reply:
Yes, this is probably a better way